.github/workflows/tests.yml

```diff
@@ -229,11 +229,27 @@ jobs:
   #         run: cargo test
 
   web:
-    name: Web
+    name: Web ${{ matrix.rust.description }}
     runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        rust:
+          - { version: stable, atomics: false }
+          - {
+              description: with Atomics,
+              atomics: true,
+              version: nightly,
+              components: rust-src,
+              flags: '-Ctarget-feature=+atomics,+bulk-memory',
+              args: '-Zbuild-std=panic_abort,std',
+            }
```

Comment:
I think we can make this `matrix` more readable by doing:
```yaml
- {
    description: Web
    version: stable
    flags: '-Dwarnings --cfg getrandom_backend="wasm_js"'
    args: '--features=std'
  }
- {
    description: Web with Atomics
    version: nightly
    flags: '-Dwarnings --cfg getrandom_backend="wasm_js" -Ctarget-feature=+atomics,+bulk-memory'
    args: '--features=std -Zbuild-std=panic_abort,std',
  }
```

And then having simpler YAML below:
```yaml
name: ${{ matrix.rust.description }}
```
```yaml
env:
  RUSTFLAGS: ${{ matrix.rust.flags }}
```
```yaml
run:
  wasm-pack test --node -- ${{ matrix.rust.args }}
```

This makes it way easier to tell at a glance what's going on and which args/flags interact.

Reply:
Done.